src/index.html

```diff
@@ -11,7 +11,181 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="field">
+        <legend>
+          Personal information
+        </legend>
+        <label for="sname" class="block">
+          Surname:
+          <input
+          type="text"
+          name="sname"
+          id="sname"
+          autocomplete="off"
+          required
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
            autocomplete="off"
            required
          >
```
<img width="979" alt="Screenshot 2022-11-11 at 19 15 56" src="https://user-images.githubusercontent.com/104769168/201394098-b85c7aa0-d4fe-4e95-abda-ef3a1017a880.png">

Reply:
fix the code style according to the checklist all over the page